packages/opencensus-core/src/trace/model/tracer.ts

```diff
@@ -14,48 +14,28 @@
  * limitations under the License.
  */
 
-import * as uuid from 'uuid';
-import * as logger from '../../common/console-logger';
-import * as loggerTypes from '../../common/types';
 import * as cls from '../../internal/cls';
-import * as configTypes from '../config/types';
-import {TraceParams} from '../config/types';
-import {noopPropagation} from '../propagation/noop-propagation';
-import {Propagation} from '../propagation/types';
-import {DEFAULT_SAMPLING_RATE, SamplerBuilder, TraceParamsBuilder} from '../sampler/sampler';
-import * as samplerTypes from '../sampler/types';
-import {NoRecordRootSpan} from './no-record/no-record-root-span';
+
 import {NoRecordSpan} from './no-record/no-record-span';
-import {RootSpan} from './root-span';
+import {CoreTracerBase} from './tracer-base';
 import * as types from './types';
 
 /**
- * This class represent a tracer.
+ * This class represents a tracer with Cointunues Local Storage (CLS).
+ *
+ * CLS helps keep tracking the root span over function calls automatically.
```

Comment:
Can we add ->  `It is capable of storing, propagating and retrieving arbitrary continuation-local data (also called "context").`

Reply:
Added